### DISCUSSION TITLE
Xamarin.Firebase.iOS.CloudMessaging bindings for Push using in Shiny

### DISCUSSION THREAD

**Discussion by kellertimo** [shinyorg/shiny#1423] [Q&A]
Since Shiny.Net also has a dependency to Xamarin.Firebase.iOS.CloudMessaging (according to Nuget) I have the following question:

Firebase notifications will not work anymore because it requires iOS SDK version >= 10 after 20.6.24

Any idea what to do here besides asking Microsoft to update the bindings?

There is also a discussion here: https://github.com/dotnet/maui/discussions/20359

**Comment by aritchie** [comment]
You could always update the bindings yourself.  It is likely what I will do if Microsoft doesn't.  I don't use firebase for all platforms in my apps, so this hasn't been a priority for me.

**Reply by kpespisa** [reply]
New information has come to light since this question was asked, most of which is captured in the dotnet maui discussion linked above. I'm curious if the plan will be to adopt the slim binding approach discussed there? (https://github.com/Redth/DotNet.Platform.SlimBindings)
